Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 MAINTAINER Rederly
 
 WORKDIR /usr/app
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
What do these do?
Why are we setting TZ to new york?

Reply:
https://anonoz.github.io/tech/2020/04/24/docker-build-stuck-tzdata.html

This is temporary - an end-run around getting 20.04 to build in docker

Reply:
Gotcha, putting that in a comment in the dockerfile would be helpful in case we forget in the future